<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>636</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>821</width>
    <height>636</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Quaver Metadata Editor</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListView" name="mapList"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="formAndActions">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="titleLabel">
           <property name="text">
            <string>Title</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="MapStringEdit" name="titleEdit">
           <property name="placeholderText">
            <string>name of the mapset...</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="tagsLabel">
           <property name="text">
            <string>Tags</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="sourceLabel">
           <property name="text">
            <string>Source</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="ListField" name="sourceEdit" native="true"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="creatorLabel">
           <property name="text">
            <string>Creator</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="MapStringEdit" name="creatorEdit">
           <property name="placeholderText">
            <string>creator of the mapset...</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="previewTimeLabel">
           <property name="text">
            <string>Preview Time</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="MapStringEdit" name="previewTimeEdit">
           <property name="inputMask">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="artistLabel">
           <property name="text">
            <string>Artist</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="MapStringEdit" name="artistEdit"/>
         </item>
         <item row="3" column="1">
          <widget class="ListField" name="tagsEdit" native="true"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="backgroundLabel">
           <property name="text">
            <string>Background</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="FileField" name="backgroundEdit" native="true"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="bannerLabel">
           <property name="text">
            <string>Banner</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="FileField" name="bannerEdit" native="true"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="audioLabel">
           <property name="text">
            <string>Audio</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="FileField" name="audioEdit" native="true"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="bottomToolbar">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="save">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MapStringEdit</class>
   <extends>QLineEdit</extends>
   <header>MapStringEdit.h</header>
  </customwidget>
  <customwidget>
   <class>ListField</class>
   <extends>QWidget</extends>
   <header>ListField.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FileField</class>
   <extends>QWidget</extends>
   <header>FileField.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
